app/src/main/res/values/strings.xml

```diff
@@ -129,4 +129,8 @@
     <string name="action_cancel">Cancel</string>
     <string name="msg_delete_comment">Delete Comment </string>
     <string name="title_confirm_delete_comment">Are you sure to delete this comment ?</string>
+
+    <string name="title_option_post">OptionPost</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_option_post` appears to be unused
 Unused resources make applications larger and slow down builds.